Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -259,9 +259,8 @@ - (CGSize)_sizeWithAttributes:(NSDictionary<NSString*, id>*)attributes constrain
         size.width = std::numeric_limits<CGFloat>::max();
     }
 
-    if (size.height == 0.0) {
-        size.height = std::numeric_limits<CGFloat>::max();
-    }
+    // Return the height that the text actually requires rather than the constrained size
+    size.height = std::numeric_limits<CGFloat>::max();
```

Comment:
With this change the introduction label in Islandwood house is being scaled properly

---
In reply to: [99943020](https://github.com/Microsoft/WinObjC/pull/1917#discussion_r99943020) [](ancestors = 99943020,99942712,99671953)